src/plots/cartesian/dragbox.js

```diff
@@ -135,6 +159,8 @@ module.exports = function dragBox(gd, plotinfo, x, y, w, h, ns, ew) {
                 clearSelect(zoomlayer);
             }
             else if(isSelectOrLasso(dragModeNow)) {
+                dragOptions.xaxes = xa;
+                dragOptions.yaxes = ya;
```

Comment:
Fixing what I broke in https://github.com/plotly/plotly.js/pull/1522/commits/98900a27b411e34b2447217abb9ba6ea87baaa48#diff-1bd3dd52cddb8638876bbe17cef28a0dL105 when I consolidated to use `recomputeAxisLists` for the initial settings too (when `dragOptions` isn't yet available, and I didn't see anyone using the `xaxes/yaxes` properties. But it turns out it was subtly broken before anyway, as the version in `recomputeAxisLists` used the wrong names (`dragoptions.xa` instead of `dragoptions.xaxes`).

`dragbox.js` is still a bit of a mess, but... I think it's a little better now.